src/features/vars-schema/hooks/vars-update-trigger.ts

```diff
@@ -9,14 +10,21 @@ interface CustomObject {
 	affectedDevices?: number[];
 }
 
+// See: https://www.postgresql.org/docs/13/protocol-message-formats.html#:~:text=The%20number%20of%20parameter%20values%20that%20follow
+const MAX_SAFE_SQL_BINDS = Math.pow(2, 16) - 1;
```

Comment:
You'll probably want to chunk slightly below the limit to allow some binds for other things like computed terms/permissions/etc

Reply:
@Page- They way I was thinking about it, was that in such a case the test cases would let us know, but I obviously forgot the case that the computed term might be bC only.
Will drop it by 100 to give it some extra room and we can change it later if we need to.

Reply:
Oh I should also switch it to use the exponential operator `**` !!